source/backend/api/Helpers/Middleware/LogRequestMiddleware.cs

```diff
@@ -67,8 +67,8 @@ public async Task Invoke(HttpContext context)
                     _logger.LogInformation($"HTTP Request {context.Request.Method} user:{context.User.GetDisplayName()} {context.Request.Scheme}://{context.Request.Host}{context.Request.Path}{context.Request.QueryString}");
                 }
 
-                _logger.LogDebug($"HTTP Request {context.Request.Method} user:{context.User.GetDisplayName()} {context.Request.Scheme}://{context.Request.Host}{context.Request.Path}{context.Request.QueryString}");
-                _logger.LogTrace(string.IsNullOrEmpty(body) ? string.Empty : $"{System.Environment.NewLine}Body: {body}");
+                _logger.LogDebug("HTTP Request {context.Request.Method} user:{context.User.GetDisplayName()} {context.Request.Scheme}://{context.Request.Host}{context.Request.Path}{context.Request.QueryString}", context.Request.Method, context.User.GetDisplayName(), context.Request.Scheme, context.Request.Host, context.Request.Path, context.Request.QueryString);
```

Comment:
evidently this is the preferred format for log messages, to make it easier to search for related log messages.